DNN Platform/Components/ClientDependency/Source/FileRegistration/Providers/BaseFileRegistrationProvider.cs

```diff
@@ -461,11 +490,11 @@ private string AppendVersion(string url, HttpContextBase http)
             {
                 //sort both the js and css dependencies properly
 
-                var jsDependencies = DependencySorter.SortItems(DependencySorter.FilterDependencies(

```

Comment:
This uses the piece above for framework deduplication

Reply:
Code restored.